src/Prelude/Linear.hs

```diff
@@ -72,7 +72,9 @@ maybe :: b -> (a ->. b) -> Maybe a ->. b
 maybe x _ Nothing = x
 maybe _ f (Just y) = f y
 
--- XXX: temporary
+-- XXX: temporary: with multiplicity polymorphism functions expecting a
+-- non-linear arrow would allow a linear arrow passed, so this would be
```

Comment:
Well, not always. But manually eta-expanding is probably not very costly anyway.

Reply:
This is true, but I think
```haskell
  dimap f g (Kleisli h) = Kleisli (\x -> forget g Prelude.<$> h (f x))
```
is clearer than
```haskell
  dimap f g (Kleisli h) = Kleisli (\x -> (\y -> g y) Prelude.<$> h (f x))
```
to read.

Reply:
Which is a use case which would, indeed, disapear with polymorphism.